src/Compilers/Test/Core/Compilation/OperationTreeVerifier.cs

```diff
@@ -57,9 +57,9 @@ public static void Verify(string expectedOperationTree, string actualOperationTr
         {
             char[] newLineChars = Environment.NewLine.ToCharArray();
             string actual = actualOperationTree.Trim(newLineChars);
-            actual = actual.Replace("\"", "\"\"");
+            actual = actual.Replace("\"", "\"\"").Replace(" \n", "\n").Replace(" \r", "\r");
```

Comment:
>Replace(" \n", "\n").Replace(" \r", "\r") [](http://example.com/codeflow?start=50&length=41)

Just curious: what required the changes to `Verify()`?

Reply:
I've been annoyed for a long time about the extra space output by the IOperation tests. This PR has a number of IOperation tests where I ran into this issue. 
This change allows the space to be ignored in comparison and removed in "actual".